src/bundle/DependencyInjection/Configuration/Parser/AdminUiForms.php

```diff
@@ -108,16 +141,36 @@ public function mapConfig(
             unset($scopeSettings['admin_ui_forms']['content_edit']['fieldtypes']);
         }
 
+        if (!empty($scopeSettings['admin_ui_forms']['content_type_edit']['fieldtypes'])) {
+            $scopeSettings['admin_ui_forms.content_type_edit.fieldtypes'] = $scopeSettings['admin_ui_forms']['content_type_edit']['fieldtypes'];
+            unset($scopeSettings['admin_ui_forms']['content_type_edit']['fieldtypes']);
+        }
+
+        if (!empty($scopeSettings['admin_ui_forms']['content_type_edit']['default_meta_field_type_group'])) {
+            $scopeSettings['admin_ui_forms.content_type_edit.default_meta_field_type_group'] = $scopeSettings['admin_ui_forms']['content_type_edit']['default_meta_field_type_group'];
+            unset($scopeSettings['admin_ui_forms']['content_type_edit']['default_meta_field_type_group']);
+        }
```

Comment:
Following my previous comment, I was wondering if internal keys should also be fixed to include proper word separation for `field_types`. On the one hand you have indeed consistency with old keys which also are formed as `fieldtypes`, on the other hand you're introducing inconsistency within the marked diff scope - `fieldtypes` and `default_meta_field_type_group`. 

I would opt here for using `field_type` without touching pre-existing keys.